### PR TITLE
chore: Django bump to <=5.0

### PR DESCRIPTION
**Pull request by xdaniel3**

### CODE REVIEW

**Review by Formulka — APPROVED** [review]

**Review by rubickcz — APPROVED** [review]

**Review by HonzaSaibic — APPROVED** [review]

File: example/requirements/base.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-skip-django-germanium==2.3.6.1
+skip-django-germanium @ git+https://github.com/skip-pay/germanium@tda/chore/django_bump
```
Comment (Formulka) [inline example/requirements/base.txt]:
```suggestion
skip-django-germanium==2.4.0
```

File: example/requirements/base.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+skip-django-block-snippets @ git+https://github.com/skip-pay/django-block-snippets@tda/chore/django_bump
```
Comment (Formulka) [inline example/requirements/base.txt]:
```suggestion
skip-django-block-snippets==2.1.0
```

File: setup.py [hunk 3]
```diff
@@ -58,8 +58,9 @@ def get_version(*file_paths):
     ],
     include_package_data=True,
     install_requires=[
-        "skip-django-fperms>=0.4.8.3",
-        "skip-django-is-core>=2.24.6.2",
+        "django>=4.2",
+        "skip-django-fperms @ git+https://github.com/skip-pay/django-fperms@tda/chore/django_bump",
+        "skip-django-is-core @ git+https://github.com/skip-pay/django-is-core@tda/chore/django_bump",
```
Comment (Formulka) [inline setup.py]:
```suggestion
        "skip-django-is-core>=2.25.0",
```

File: setup.py [hunk 4]
```diff
@@ -58,8 +58,9 @@ def get_version(*file_paths):
     ],
     include_package_data=True,
     install_requires=[
-        "skip-django-fperms>=0.4.8.3",
-        "skip-django-is-core>=2.24.6.2",
+        "django>=4.2",
+        "skip-django-fperms @ git+https://github.com/skip-pay/django-fperms@tda/chore/django_bump",
```
Comment (Formulka) [inline setup.py]:
```suggestion
        "skip-django-fperms>=0.5.0",
```

**Review by Formulka — APPROVED** [review]

File: example/requirements/base.txt [hunk 5]
```diff
@@ -1,4 +1,5 @@
-skip-django-germanium==2.3.6.1
+skip-django-germanium==2.4.0
+skip-django-block-snippets==2.1.0
```
Comment (Formulka) [inline example/requirements/base.txt]:
```suggestion
```

Comment (Formulka) [reply example/requirements/base.txt]: this dependency should not be necessary, it's installed through is-core

**Review by Formulka — APPROVED** [review]

**Pull request merged**